cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -19,6 +19,58 @@
 
   <!-- General Configuration -->
 
+  <property>
+    <name>ai.vertexai.project.id</name>
+    <value></value>
+    <description>
+      Name of the project.
+    </description>
+  </property>
+
+  <property>
+    <name>ai.vertexai.location</name>
+    <value></value>
+    <description>
+      Name of the location.
+    </description>
+  </property>
+
+  <property>
+    <name>ai.vertexai.model</name>
+    <value>gemini-1.5-pro</value>
+    <description>
+      Name of the model to be used for availing AI service through VertexAI.
+    </description>
+  </property>
+  <property>
+    <name>ai.vertexai.prompts.markdown.pipeline.summary</name>
+    <value>
+      Summarize the above provided Cloud Data Fusion pipeline JSON in a concise and informative Markdown format. The summary should include:
```

Comment:
There is no notion of Cloud Data Fusion in CDAP OSS.

Either replace this part with: `CDAP pipeline`.

Or replace the entire prompt to simply: `Summarize the CDAP data pipeline`,